Makefile

```diff
@@ -156,8 +156,8 @@ gen-k8s-check: $(apis_sources)
 	./hack/verify-codegen.sh
 	touch $@
 
-bump-all:
-	./hack/components/bump-all.sh
+bump-components:
```

Comment:
I think we don't need bump-components.sh just small Makefile target is enough.

```Makefile
bump-*:
    CNAO_VERSION=${VERSION} ./hack/components/bump-$@.sh
bump-all: bump-kubemacpool bump-kubernets-nmstate bump ....
```

Or even more exlicit with target per component

Reply:
sure, do we mind if we use the regular go folder and not the temp as in the [start](https://github.com/kubevirt/cluster-network-addons-operator/pull/449/files#diff-a15786f49370ff37a2c3d11d804b3eeeR5) of the bump_components.sh: